docs/dev_guide/nc_nsfs_iam_developer_doc.md

```diff
@@ -51,4 +51,13 @@ Note: Currently we clean the cache after update, but it happens for the specific
 1. Use the root account credentials to create a user: `nc-user-1-iam iam create-user --user-name <username>` (You should see the config file in under the accounts directory).
 2. Use the root account credentials to create access keys for the user:(first time): `nc-user-1-iam iam create-access-key --user-name <username>` (You should see the first symbolic link in under the access_keys directory).
 3. Use the root account credentials to create access keys for the user (second time): `nc-user-1-iam iam create-access-key --user-name <username>` (You should see the second symbolic link in under the access_keys directory).
-4. Update the username: `nc-user-1-iam iam update-user --user-name <username> --new-user-name <new-username>` (You should see the following changes: config file name updated, symlinks updated according to the current config).
\ No newline at end of file
+4. Update the username: `nc-user-1-iam iam update-user --user-name <username> --new-user-name <new-username>` (You should see the following changes: config file name updated, symlinks updated according to the current config).
+
+#### Create root account using the IAM API (requesting account is root accounts manager):
+1. Create the root accounts manager with the CLI:
+`sudo node src/cmd/manage_nsfs account add --name <name>> --new_buckets_path /tmp/nsfs_root1 --access_key <access-key> --secret_key <secret-key> --uid <uid> --gid <gid> --iam_operate_on_root_account`.
```

Comment:
@shirady @guymguym WDYT about `--root_accounts_manager` or `--iam_root_accounts_manager`?

Reply:
@romayalon Our concern when we chose `--iam_operate_on_root_account` was not having a double meaning, and since the term "manager" doesn't have a specific meaning in IAM API and doesn't really say what it allows, it might sound like this account owns the other accounts or their buckets, while it doesn't.

So in the schema I would keep it as is, but in the CLI we can consider getting more external feedback if people prefer to use `--iam_root_accounts_manager` and understand the nuances that it has.

Reply:
At this point, I will not change the name, and when we receive the feedback I can change it in the future.